### DISCUSSION TITLE
How can I translate the player sheet to another language?

### DISCUSSION THREAD

**Discussion by IgorAK25** [deltagreen-foundryvtt/delta-green-foundry-vtt-system#52] [Q&A] [answered]
I want to translate the sheet to Portuguese, so my players can better understand it. I have no prior knowledge on how to code, can I use babele or something like it?

**Comment by TheLastScrub** [comment, marked as answer]
Hey @IgorAK25, you don't need to know how to code in order to help localize the system.  You just need to grab a copy of one of the latest language packs, e.g. en.json from here:

https://github.com/TheLastScrub/delta-green-foundry-vtt-system-unofficial/tree/master/lang

Make a copy of the file and name it according to your language (I think Portuguese is "pt") and change all the values from English to whatever your target language is:

![image](https://user-images.githubusercontent.com/67403321/158030055-0c64d686-5420-40c8-8845-deb08343fb6d.png)

Basically just fill it out with Portuguese and send it back to me.

You can test how your translation is looking by browsing to the Delta Green system in your local Foundry install and placing it next to the other language files, and then updating the system.json file to include your new language pack.  See screenshot below for where to add it in the JSON:

![image](https://user-images.githubusercontent.com/67403321/158030193-bdd5581d-c440-418b-af19-90ce13971e47.png)

NOTE! Any changes you make to your local system install will GET DELETED if you update your Delta Green system version!  So once you are happy with your language pack, send it to me and I will include it in the system so anyone can use it.

Keep in mind though that if I update the system, I won't necessarily keep the non-English language packs up to date. You can help update them in the future if needed though.

Alternatively, if you are comfortable with GitHub, you can also pull a copy of the system down, make your changes, and then just make a pull request to get everything added/updated.

**Reply by IgorAK25** [reply]
Can you send me your discord so I can send you the file?

**Reply by Karvattatus** [reply]
Hello, @TheLastScrub, I'm getting a French version ready and I'd like to get in touch with you in this regards. Thanks!
